frontend/smart-kickers-game/src/apis/goalManipulation.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const serverUrl = 'http://localhost:3006';
```

Comment:
This is great! Would be even better if implemented in #67 (ofc not required in this PR) and here moved upwards to a separate file, so that other (possible) files in `apis` directory can share it.

Reply:
We will change in later in another PR.